src/view/components/extensionView.jsx

```diff
@@ -135,7 +135,7 @@ const ExtensionView = ({
       try {
         return await viewRegistrationRef.current.getSettings({
```

Comment:
What if we did not call getSettings at all if the formik values are undefined?

Reply:
@jonsnyder I changed the initial PR. Now getSettings will return `{}` when no Formik values are present.